.github/renovate.json

```diff
@@ -28,6 +28,7 @@
       "automerge": true
     }
   ],
+  "ignoreDeps": ["moment-timezone"],
```

Comment:
```suggestion
  "ignoreDeps": ["moment-timezone"],
  "__Open_edX_ignoreDeps_reasons__": {
    "__note__": "When you add a package to ignoreDeps, add a comment to this object, using the package name as the key, and the comment as the value.",
    "moment-timezone": "Broke legacy programs dashboard, which is deprecated and soon to be removed. See https://github.com/openedx/edx-platform/pull/34928"
  },
```

Reply:
as discussed in Slack -- if Renovate complains about the fake JSON field, I'll fix forward

Reply:
Looks like Renovate doesn't like the extra fields: https://github.com/openedx/edx-platform/issues/34940

Reply:
On it